crates/nargo_cli/Cargo.toml

```diff
@@ -37,8 +37,7 @@ termcolor = "1.1.2"
 color-eyre = "0.6.2"
 
 # Backends
-aztec_backend = { optional = true, package = "barretenberg_static_lib", git = "https://github.com/noir-lang/aztec_backend", rev = "e3d4504f15e1295e637c4da80b1d08c87c267c45" }
-aztec_wasm_backend = { optional = true, package = "barretenberg_wasm", git = "https://github.com/noir-lang/aztec_backend", rev = "e3d4504f15e1295e637c4da80b1d08c87c267c45" }
+acvm-backend-barretenberg = { git = "https://github.com/noir-lang/aztec_backend", rev = "ba1d0d61b94de91b15044d97608907c21bfb5299", default-features=false }
```

Comment:
Why false for default features?

Reply:
This would result in us always using the native backend (and so not being able to switch to wasm)

Reply:
Can we not just build with a feature flag to swim to wasm?

Reply:
This is what we do with the lines

```toml
plonk_bn254 = ["acvm-backend-barretenberg/native"]
plonk_bn254_wasm = ["acvm-backend-barretenberg/wasm"]
```

The issue is that you can't turn off a feature flag once it's been turned on in dependency tree (except when you're turning off all default features) so we'd either have `native` or `native+wasm` (and trigger a compilation error)

https://stackoverflow.com/questions/39734321/is-it-possible-to-disable-a-single-default-feature-in-cargo

Reply:
Ah got it. Thanks for explaining